app/scripts/TiledPixiTrack.js

```diff
@@ -518,7 +518,7 @@ class TiledPixiTrack extends PixiTrack {
     return this.scale.maxRawValue;
   }
 
-  initTile(/* tile */) {
+  initTile(tile) {
```

Comment:
What's going on here?

Reply:
It was commented out to shut up the linter because `tile` wasn't being used before. Now it's being used.